File: beecrowd/principiante/07-par_impar_positivo_negativo/paim.cpp
#include <stdio.h>

int main() {
  int n1,n2,n3,n4,n5; scanf("%d %d %d %d %d", &n1, &n2, &n3, &n4, &n5);
  int valores[5] = {n1,n2,n3,n4,n5};
  int par=0, imp=0, pos=0, neg=0;
  for (int i=0; i<5; i++) {
    if (valores[i] > 0) {
      pos++;
    } else if (valores[i] < 0) {
      neg++;
    }

    if (valores[i]%2==0) {
      par++;
    } else {
      imp++;
    }
  }

  printf("%d valor(es) par(es)\n", par);
  printf("%d valor(es) impar(es)\n", imp);
  printf("%d valor(es) positivo(s)\n", pos);
  printf("%d valor(es) negativo(s)\n", neg);
  
  return 0;
}

File: beecrowd/principiante/07-par_impar_positivo_negativo/paim.py
a=[]
pares=0
impares=0
positivos=0
negativos=0
for i in range(5):
    a.append(int(input()))
    if a[i]%2 == 0:
        pares+=1
    else:
        impares+=1

    if a[i] < 0:
        negativos+=1
    elif a[i] > 0:
        positivos+=1
print(f'{pares} valor(es) par(es)')
print(f'{impares} valor(es) impar(es)')
print(f'{positivos} valor(es) positivo(s)')
print(f'{negativos} valor(es) negativo(s)')
